core/src/main/scala/slamdata/SbtSlamDataBase.scala

```diff
@@ -426,12 +438,72 @@ abstract class SbtSlamDataBase extends AutoPlugin {
       },
 
       resolvers ++= {
-        if (publishAsOSSProject.value)
+        if (!publishAsOSSProject.value)
           Seq(Resolver.bintrayRepo("slamdata-inc", "maven-private"))
         else
           Seq.empty
-      }
-    )
+      },
 
+      trickleCreatePullRequest := {
+        val prior = trickleCreatePullRequest.value
+        val log = sLog.value
+
+        { (repo: OutdatedRepository) =>
+          import cats.effect.{ContextShift, IO}
+
+          import github4s.Github
+          import github4s.domain.NewPullRequestData
+
+          import java.nio.file.Files
+          import scala.concurrent.ExecutionContext.Implicits.global
+
+          implicit val cs: ContextShift[IO] = IO.contextShift(global)
+
+          prior(repo)
+
+          val authenticated = {
+            val uri = new URI(repo.url)
+            s"${uri.getScheme}://${sys.env("GITHUB_ACTOR")}:${sys.env("GITHUB_TOKEN")}@${uri.getHost}${uri.getPath}"
+          }
+
+          val dir = Files.createTempDirectory("sbt-slamdata")
+          if (runWithLogger(s"git clone --depth 1 $authenticated ${dir.toFile.getPath}", log) != 0) {
+            sys.error("git-clone exited with error")
+          }
+
+          val branchName = s"trickle/${System.currentTimeMillis}"
+          if (runWithLogger(s"cd $dir; git checkout -b $branchName", log) != 0) {
+            sys.error("git-checkout exited with error")
+          }
+
+          val managed = ManagedVersions(dir.resolve(VersionsPath))
+          repo.updates foreach {
+            case ModuleUpdateData(_, _, newRevision, repo, _) =>
```

Comment:
I'm not fond of `repo` shadowing here. I'm opposed to everything that makes me double take.